crates/matrix-sdk-base/src/store/mod.rs

```diff
@@ -319,6 +270,175 @@ impl Deref for Store {
     }
 }
 
+/// An observable map.
```

Comment:
Should this go to its own file?

Reply:
I was wondering the same. Since we are too thinking that, I'll extract this into its own module.